environments/staging/app-processes.yml

```
@@ -146,7 +146,7 @@ pillows:
       processor_chunk_size: 1
       dedicated_migration_process: True
     demo-case-pillow:
-      # add include_demo_topic: True to case-pillow if you remove this to drain the demo topic
+      # set RUN_DEMO_CASE_PILLOW to False if you remove this to drain the demo topic
```

You wouldn't "remove this to drain the demo topic", right? Suggest rephrase
```suggestion
      # set RUN_DEMO_CASE_PILLOW to False to drain the demo topic if you remove this
```
Or maybe I misunderstood how this works?